feedback-module/src/assets/styling_classnames.js

```diff
@@ -16,4 +16,4 @@ export const LABEL_STYLE =
   "text-primary mobile-lg:font-sans-2xl font-sans-lg maxw-none margin-y-3";
```

Comment:
Why are there two separate styling_classnames files?

Reply:
hmmm looks like i accidentally duplicated it, thanks for catching that!